.gitmodules

```diff
@@ -10,10 +10,14 @@
 	path = lemmy-translations
 	url = https://github.com/lemmynet/lemmy-translations
 	branch = main
-[submodule "lemmy-js-client"]
-	path = lemmy-js-client
+[submodule "lemmy-js-client-latest"]
+	path = lemmy-js-client-latest
 	url = https://github.com/LemmyNet/lemmy-js-client
 	branch = release/v0.19
+[submodule "lemmy-js-client-next"]
+	path = lemmy-js-client-next
+	url = https://github.com/LemmyNet/lemmy-js-client
+	branch = tsoa_1
```

Comment:
I've added serving up multiple API docs at the same time now. 

If you have `joinlemmy-site` on your system, I highly recommend deleting the folder and recloning, because working with changing gitsubmodules is a nightmare.